File: app/src/main/java/ximenapps/com/br/marvelbynatan/service/CharactersService.java
package ximenapps.com.br.marvelbynatan.service;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import ximenapps.com.br.marvelbynatan.model.CharactersDataWrapper;

/**
 * Created by Natan on 04/08/2016.
 */
public interface CharactersService {

    @GET("characters")
    Call<CharactersDataWrapper> listCharacters();
}

File: app/src/main/java/ximenapps/com/br/marvelbynatan/service/CharactersService.kt
package ximenapps.com.br.marvelbynatan.service

import io.reactivex.rxjava3.core.Single
import retrofit2.http.GET
import ximenapps.com.br.marvelbynatan.model.CharactersDataWrapper

interface CharactersService {
    @GET("characters")
    fun listCharacters(): Single<CharactersDataWrapper>
}